sc/social/like/upgrades/v3047/configure.zcml

```diff
@@ -3,8 +3,8 @@
     xmlns:genericsetup="http://namespaces.zope.org/genericsetup">
 
   <genericsetup:upgradeSteps
-      source="3046"
-      destination="3047"
+      source="3047"
```

Comment:
this is wrong; you need to register a new upgrade step.

Reply:
ok